src/app/features/task-repeat-cfg/store/task-repeat-cfg.effects.ts

```diff
@@ -58,6 +66,131 @@
     { dispatch: false },
   );
 
+  /**
+   * Updates the repeatCfg of a task, if the task was updated.
+   */
+  updateRepeatCfgWhenTaskUpdates: any = createEffect(
+    () =>
+      this._actions$.pipe(
+        ofType(updateTask),
+        tap(async (aAction) => {
```

Comment:
The 'await' inside the 'tap' operator may lead to unexpected behavior. It's better to use 'mergeMap' or 'concatMap' to handle asynchronous operations in an Observable chain. [important]